src/Model/Behavior/Strategy/TuupolaStrategy.php

```diff
@@ -0,0 +1,51 @@
+<?php
+declare(strict_types=1);
+
+namespace Muffin\Obfuscate\Model\Behavior\Strategy;
+
+use InvalidArgumentException;
+use Tuupola\Base62;
+
+/**
+ * Class TuupolaStrategy
+ */
+class TuupolaStrategy implements StrategyInterface
```

Comment:
We should probably name this new class `Base64Strategy` to be future proof. Then if needed the underlying lib can be changed without renaming/deprecating the class.